src/pages/ClipboardGuide.jsx

```
@@ -0,0 +1,30 @@
+                <img src={snapshort1} alt='snapshot1'
+                className="flex justify-center items-center rounded-t-xl "/>
+            </div>
+
+            <div className="text-left mt-5 backdrop-blur-sm">
+                A clipboard search feature that lets users quickly find any saved clipboard entry by typing keywords. It filters items in real-time, showing relevant results with details like timestamp, size, and usage, enabling fast retrieval and efficient clipboard management.                </div>
+            <div className="m-5 px-10 pt-10 rounded-t-xl bg-[#929292]">
+                <img src={snapshot2} alt='snapshot2'
+                className="flex justify-center items-center rounded-t-xl "/>
+            </div>
+
+            <Link to='/OnlineSearchGuide'><button className="relative bottom-3 left-170 flex flex-row justify-center py-1 px-3 bg-black rounded-md text-center text-white ">next</button></Link>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)